webui/react/src/pages/ResourcePool/NodeElement.tsx

```diff
@@ -0,0 +1,83 @@
+import Tooltip from 'hew/Tooltip';
+import React, { PropsWithChildren, useCallback, useEffect, useMemo, useRef, useState } from 'react';
+
+import { Resource, SlotsRecord } from 'types';
+
+import css from './NodeElement.module.scss';
+
+export type SimplifiedNode = { container?: { id: string } };
```

Comment:
if its a part of `Resource` type, can we use utility type instead of hard coding?